app/services/appointments/new_appointment_mail_to_freelancer.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Appointments
+  class NewAppointmentMailToFreelancer
+    def self.call(appointment)
+      return unless appointment.persisted?
+
+      FreelancerAppointmentMailer.new_appointment_mail_to_freelancer(appointment).deliver_now
```

Comment:
Can't we just move this to the [pay appointment file](app/services/appointments/pay_appointment.rb)? It's not an issue since it's just one line and appointment.persisted check is already built into the pay_appointment method since it's using an Active Record transaction.